src/SWP/Bundle/ContentBundle/Tests/ArticleLoaderTest.php

```diff
@@ -53,11 +49,11 @@ public function testFindNewArticle()
         $this->assertTrue($articleLoader->isSupported('articles'));
         $this->assertFalse($articleLoader->isSupported('items'));
 
-        $article = $articleLoader->load('article', ['contentPath' => '/swp/default/content/test-article']);
+        $article = $articleLoader->load('article', ['contentPath' => '/swp/123456/123abc/content/test-article']);
```

Comment:
Can you put the path in a variable and reuse it, instead of copy and pasting the value? Then you would only ever have to change it once.